packages/react-core/src/hooks/useDropZone.ts

```diff
@@ -102,20 +102,11 @@ export default function useDropZone({
         onDropComplete({ acceptedFiles, rejectedFiles });
       }
       return;
```

Comment:
Nit: I think this `return` is now unneeded?

Reply:
+1

Reply:
Ah, yep! Nice catch. Removed